content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -102,6 +103,73 @@ model UserConfig {
 }
 ```
 
+## How to apply your schema changes with Prisma Migrate and <inlineCode>db push</inlineCode>
+
+<Admonition type="warning">
+
+Multi-schema support for Prisma Migrate and `db push` is currently only available with the PostgreSQL connector.
+
+</Admonition>
+
+You can use Prisma Migrate or `db push` to apply changes to a Prisma schema with multiple database schemas.
+
+As an example, add a `Profile` model in the `base` schema of the blog post model above:
+
+```prisma file=schema.prisma highlight=4,9-16;add
+model User {
+  id      Int      @id
+  orders  Order[]
+  profile Profile?
+
+  @@schema("base")
+}
+
+model Profile {
+  id     Int    @id @default(autoincrement())
+  bio    String
+  user   User   @relation(fields: [userId], references: [id])
+  userId Int    @unique
+
+  @@schema("base")
+}
+
+model Order {
+  id      Int  @id
+  user    User @relation(fields: [id], references: [id])
+  user_id Int
+
+  @@schema("transactional")
+}
+
+enum Size {
+  Small
+  Medium
+  Large
+
+  @@schema("transactional")
+}
+```
+
+You can then apply this schema change to your database. For example, you can use `migrate dev` to create and apply your schema changes as a migration:
+
+```terminal
+npx prisma migrate dev --name add_profile
+```
+
+Note that if you move a model or enum from one schema to another, Prisma will delete the model or enum from the source schema and create a new one in the target schema.
+
+## How to introspect an existing database with multiple database schemas
+
+<!-- TODO: check if need admonition about SQL Server -->
```

Comment:
TODO

Reply:
We're still at the same point as last week, i.e. not sure it will make it, but I am getting slightly more confident that it will make it in next week's release.

Reply:
(introspection only)